.pylintrc

```diff
@@ -18,6 +18,7 @@ ignore-patterns=
 #init-hook=
 
 # Use multiple processes to speed up Pylint.
+# jobs=1
```

Comment:
Looks like there is a bunch of commented out changes; just make sure to remove them before you're done.